server/public/styles.css

```diff
@@ -111,3 +111,93 @@ p {
   font-family: 'Overpass', sans-serif;
   margin-top: 1rem;
 }
+
+.modal-button-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 8rem;
+}
+
+.modal-button {
+  background-color: #16c79a;
+  border: none;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 50%;
+}
+
+.modal-container {
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  position: fixed;
+}
+
+.modal {
+  background-color: white;
+  width: 76%;
+  display: flex;
+  justify-content: space-evenly;
+  height: 200px;
+  padding-top: 4rem;
+}
+
+.fas.fa-utensils,
+.fas.fa-dumbbell {
+  font-size: 1.5rem;
+  color: black;
+}
+
+.fas.fa-utensils {
+  padding-left: 2.2rem;
+}
+
+.fas.fa-dumbbell {
+  padding-left: 2.75rem;
+}
+
```

Comment:
In general, we do not style CSS classes that we did not define. Instead add your own CSS classes, like `.workout-journal-icon`.